experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1507,18 +1507,25 @@ def _validate_rollout_version_support(self, data):
         return data
 
     def _validate_bucket_duplicates(self, data):
-        if not self.instance or not self.instance.is_rollout:
-            return data
         is_rollout = self.instance.is_rollout
-        count = NimbusExperiment.objects.filter(
+        if not self.instance or not is_rollout:
+            return data
+
+        matches = []
+        configs = self.instance.feature_configs.all()
+        for experiment in NimbusExperiment.objects.filter(
             status=NimbusExperiment.Status.LIVE,
             channel=self.instance.channel,
             application=self.instance.application,
             targeting_config_slug=self.instance.targeting_config_slug,
             is_rollout=is_rollout,
-        ).count()
+        ):
+            if experiment.feature_configs.exists():
+                for feature_config in experiment.feature_configs.all():
+                    if feature_config in configs:
+                        matches.append(experiment)
 
-        if is_rollout and count > 0:
+        if matches.__len__() > 0:
```

Comment:
```py
NimbusExperiment.objects.filter(
    ...
    feature_configs__in=self.instance.feature_configs.all()
    ...
)
```

;)

Reply:
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#in

Reply:
`__in`!!! brilliant